### PR TITLE
Correctly format scripting keywords

### PR DESCRIPTION
**Pull request by relud**
adds tests for `script/format_sql` as directories that contain `expect.sql`, and optionally `input.sql` if it differs from `expect.sql`

to avoid complex handling of `THEN`, `CASE` formatting has changed from
```
CASE
WHEN ... THEN ...
END
```
to 
```sql
WHEN
  ...
THEN
  ...
```

### CODE REVIEW

**Comment by relud** [comment]
This also adds blank lines between statements in formatted files

File: sql/telemetry/fxa_users_daily_v1/query.sql [hunk 1]
```diff
@@ -38,11 +36,9 @@ CREATE TEMP FUNCTION udf_contains_tier1_country(
       )
   )
 );
-  --
```
Comment (acmiyaguchi) [inline sql/telemetry/fxa_users_daily_v1/query.sql]: This statement looks like it was generated by `prepend_udf_usage_definitions`.

File: sql/telemetry/fxa_users_daily_v1/query.sql [hunk 2]
```diff
@@ -38,11 +36,9 @@ CREATE TEMP FUNCTION udf_contains_tier1_country(
...
-  -- This UDF is also only applicable in the context of this query.
-CREATE TEMP FUNCTION udf_contains_registration(
-  x ANY TYPE
-) AS ( --
```
Comment (acmiyaguchi) [inline sql/telemetry/fxa_users_daily_v1/query.sql]: But was this this `--` also generated by the same function? The new formatting looks much better.

Comment (relud) [reply sql/telemetry/fxa_users_daily_v1/query.sql]: i manually fixed up this formatting

File: sql/telemetry/fxa_users_daily_v1/query.sql (hunk 1 above)
Comment (relud) [reply sql/telemetry/fxa_users_daily_v1/query.sql]: yeah, it was

**Pull request merged**